src/prepare/BasePrepare.js

```diff
@@ -290,6 +320,84 @@ export default class BasePrepare
 }
 
 /**
+ * Built-in hook to find multiple textures from objects like AnimatedSprites.
+ *
+ * @private
+ * @param {PIXI.DisplayObject} item - Display object to check
+ * @param {Array<*>} queue - Collection of items to upload
+ * @return {boolean} if a PIXI.Texture object was found.
+ */
+function findMultipleBaseTextures(item, queue)
+{
+    // Objects with mutliple textures
+    if (item && item._textures && item._textures.length)
+    {
+        for (let i = 0; i < item._textures.length; i++)
+        {
+            const baseTexture = item._textures[i].baseTexture;
```

Comment:
It would probably be safer to check that `item._textures[i]` is an instanceof Texture, much like the `findTexture` method below. Also, should only `return true` if something is found.